components/activity_and_insights/insights/insights.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {memo, useEffect} from 'react';
+import {useDispatch} from 'react-redux';
+
+import {selectChannel} from 'mattermost-redux/actions/channels';
+
+import TopChannels from './top_channels/top_channels';
+
+import './../activity_and_insights.scss';
+
+const Insights = () => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(selectChannel(''));
```

Comment:
Is this meant to have an empty string parameter?

Reply:
Yeah, threads does the same thing in order to clear the current channel. If we don't do this the previous channel you navigated from will be highlighted